sbol3/document.py

```diff
@@ -11,6 +10,7 @@
 from rdflib_jsonld import parser as jsonld_parser
 
 from . import *
+from .object import BUILDER_REGISTER
```

Comment:
This import implies that `BUILDER_REGISTER` is not available by importing sbol3. How will custom classes get access to `BUILDER_REGISTER`?

Did I miss something?

Reply:
Custom classes (and the core classes which also register their builder) are registered using the `Document.register_builder` static method.  So no change.

`BUILDER_REGISTER` need only be accessible to the `Document` class.  See this line:
https://github.com/SynBioDex/pySBOL3/blob/123-copy_methods/sbol3/document.py#L37